Makefile

```diff
@@ -64,11 +64,11 @@ analyze: FORCE
 
 backtesting-piecewise: FORCE
 	# Generates all the backtesting models piece by piece so that it can be run on a GPU enabled machine
-	python scripts/mutrans.py --backtesting-max-day `seq -s, 150 14 220` --forecast-steps 12
-	python scripts/mutrans.py --backtesting-max-day `seq -s, 220 14 500` --forecast-steps 12
-	python scripts/mutrans.py --backtesting-max-day `seq -s, 500 14 625` --forecast-steps 12
-	python scripts/mutrans.py --backtesting-max-day `seq -s, 626 14 700` --forecast-steps 12
-	python scripts/mutrans.py --backtesting-max-day `seq -s, 710 14 766` --forecast-steps 12
+	python scripts/mutrans.py --backtesting-max-day `seq -s, 150 14 220` --forecast-steps 12 --model-type reparam-localinit-nofeatures
+	python scripts/mutrans.py --backtesting-max-day `seq -s, 220 14 500` --forecast-steps 12 --model-type reparam-localinit-nofeatures
+	python scripts/mutrans.py --backtesting-max-day `seq -s, 500 14 625` --forecast-steps 12 --model-type reparam-localinit-nofeatures
+	python scripts/mutrans.py --backtesting-max-day `seq -s, 626 14 700` --forecast-steps 12 --model-type reparam-localinit-nofeatures
+	python scripts/mutrans.py --backtesting-max-day `seq -s, 710 14 766` --forecast-steps 12 --model-type reparam-localinit-nofeatures
```

Comment:
You might want to create a new makefile target called `backtesting-nofeatures` so that nicks old command still works

Reply:
thanks, added backtesting-nofeatures to the makefile